app/components/templates/cult/cult.component.tsx

```diff
@@ -0,0 +1,50 @@
+import {FC} from 'react';
+
+import {
+  BottomContainer,
+  CultSection,
+  PerksGrid,
+  TopContainer,
+  Description,
+  CarouselContainer,
+} from './cult.styles';
+import {PerkInfo} from '../../molecules/perk-info/perk-info.component';
+import {TitleSection} from '../../molecules/title-section';
+import {Carousel} from '../../organisms/carousel';
+
+type CarouselImage = {
+  id: string;
+  alt: string;
+  src: string;
+};
+
+interface Props {
+  images: CarouselImage[];
+}
```

Comment:
Why would you choose `type` for CarouselImage and `interface` for props?

Reply:
I typically feel more comfortable using custom types inside of interfaces; it just makes more sense to me.

Reply:
I see, but since you're not working alone, lets keep using interface until we have to use type for some reason.